extensions/historical_imagery/README.md

```diff
@@ -30,9 +30,9 @@ change**
 
 ### [STAC Item Properties](https://github.com/radiantearth/stac-spec/blob/v1.0.0/item-spec/item-spec.md#properties-object)
 
-| Field Name | Type         | Description                                                                                                                                                                                                                                                                                                                                                                                   | Internal Field Name | Example                | Comments        |
-| ---------- | ------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------- | ---------------------- | --------------- |
-| datetime   | string\|null | **REQUIRED**. The searchable date and time of the assets, in UTC. It is formatted according to [RFC 3339, section 5.6](https://tools.ietf.org/html/rfc3339#section-5.6) and [ISO 8601](https://www.iso.org/iso-8601-date-and-time-format.html). `null` is allowed, but requires `start_datetime` and `end_datetime` from [common metadata](common-metadata.md#date-and-time-range) to be set. | date                | `1958-01-23T00:00:00Z` | Convert to UTC. |
+| Field Name | Type         | Description                                                                                                                                                                                                                                                                                                                                         | Internal Field Name | Example                | Comments        |
+| ---------- | ------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------- | ---------------------- | --------------- |
+| datetime   | string\|null | **REQUIRED**. The searchable date and time of the assets, in UTC. It is formatted according to [RFC 3339, section 5.6](https://tools.ietf.org/html/rfc3339#section-5.6) and [ISO 8601](https://www.iso.org/iso-8601-date-and-time-format.html). `null` is allowed, but requires `start_datetime` and `end_datetime` from common metadata to be set. | date                | `1958-01-23T00:00:00Z` | Convert to UTC. |
```

Comment:
Link should have been to: https://github.com/radiantearth/stac-spec/blob/v1.0.0/item-spec/common-metadata.md#date-and-time-range